cedar-policy-core/src/est/expr.rs

```diff
@@ -735,7 +721,13 @@ impl From<ast::Expr> for Expr {
             ast::ExprKind::Lit(lit) => lit.into(),
             ast::ExprKind::Var(var) => var.into(),
             ast::ExprKind::Slot(slot) => slot.into(),
-            ast::ExprKind::Unknown(ast::Unknown { name, .. }) => Expr::unknown(name),
+            ast::ExprKind::Unknown(ast::Unknown { name, .. }) => {
```

Comment:
The natural follow-up question is if we need `ast::ExprKind::Unknown`. I'm not sure myself

Reply:
I think there are still many dependencies on `ast::ExprKind::Unknown`, don't think we should remove it for this PR